OrbitCore/CallstackData.h

```diff
@@ -61,6 +81,12 @@ class CallstackData {
     }
   }
 
+  [[nodiscard]] absl::flat_hash_map<CallstackID, std::shared_ptr<CallStack>> unique_callstacks()
```

Comment:
this returns a copy... maybe make the name reflect this? GetUniqueCallstacksCopy()?

Reply:
Done. Thanks.